src/ensembl/src/content/app/entity-viewer/state/sidebar/sampleData.ts

```diff
@@ -0,0 +1,749 @@
+      },
+      {
+        id: 'ENST00000530893.6'
+      },
+      {
+        id: 'ENST00000670614.1'
+      },
+      {
+        id: 'ENST00000671466.1'
+      },
+      {
+        id: 'ENST00000665585.1',
+        xrefs: [
+          {
+            name: 'UniParc',
+            value: 'UPI00114938A5',
+            url: ''
+          }
+        ]
+      },
+      {
+        id: 'ENST00000470094.1'
+      },
+      {
+        id: 'ENST00000666593.1'
+      },
+      {
+        id: 'ENST00000528762.1'
+      },
+      {
+        id: 'ENST00000533776.1',
+        xrefs: [
+          {
+            name: 'RNAcentral',
+            value: 'URS0000EACAD8',
+            url: ''
+          },
+          {
+            name: 'UCSC Stable ID',
+            value: 'ENST00000533776.1',
+            url: ''
+          }
+        ]
+      },
+      {
+        id: 'ENST00000614259.1'
+      }
+    ],
+    filters: {
+      transcript: {
+        sequence: [
+          {
+            value: 'Genomic sequence',
+            label: 'Genomic sequence'
+          },
+          {
+            value: 'cDNA',
+            label: 'cDNA'
+          },
+          {
+            value: 'CDS',
+            label: 'CDS'
+          },
+          {
+            value: 'Protein sequence',
+            label: 'Protein sequence'
+          }
+        ],
+        tark_url: 'http://betatark.ensembl.org/web/search/'
+      }
+    }
+  },
+  other_assemblies: [
+    {
+      species_name: 'Human',
+      assembly_name: 'GRCh37',
+      stable_id: 'ENSG00000139618'
+    }
+  ],
+  other_data_sets: [
+    { type: DataSetType.CURRENT_ASSEMBLY, value: 'GRCh38.p13' },
+    { type: DataSetType.GENE, value: 'ENSG00000139618.15' }
+  ],
+  homeologues: [
+    {
+      type: 'Gene',
+      stable_id: 'TraesCS3A02G274400'
+    },
+    {
+      type: 'Gene',
+      stable_id: 'TraesCS3B02G308100'
+    }
+  ],
+  publications: [
+    {
+      title: 'Identification of the breast cancer susceptibility gene BRCA2.',
+      description:
+        'Wooster R, Bignell G, Lancaster J, Swift S, Seal S, Mangion J, Collins N, Gregory S, Gumbs C, Micklem G, Barfoot R, Hamoudi R, Patel S, Rice C, Biggs P, Hashim Y, Smith A, Connor F, Stratton MR',
+      source: {
+        url: '',
+        value: 'Europe PMC',
+        name: ''
+      }
+    },
+    {
+      title:
+        'The complete BRCA2 gene and mutations in chromosome 13q-linked kindreds',
+      description:
+        'Tavtigian SV, Simard J, Rommens J, Couch F, Shattuck-Eidens D, Neuhausen S, Merajver S, Thorlacius S, Offit K, Stoppa-Lyonnet D, Belanger C, Bell R, Berry S, Bogden R, Chen Q, Davis T, Dumont M, Frye C, Goldar DE',
+      source: {
+        url: '',
+        value: 'Europe PMC',
+        name: ''
+      }
+    },
+    {
+      title: '',
+      description: '',
+      source: {
+        url: '',
+        value: 'NIEHS SNPs program',
+        name: 'Submitted (OCT-2003) to the EMBL/GenBank/DDBJ databases'
+      }
```

Comment:
???
This reference to a publication does not make sense. It doesn't make sense in the design either. I'll ask Andrea where she saw it, but I think it will be safest for us to exclude this reference entirely.

Reply:
Cool, I've removed it for now.